packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -484,6 +484,21 @@ export default class BackupNg {
         })
       }
       return result.id
+    } catch (error) {
+      if (
+        // xxhash is the new form consistency hashing in CH 8.1 which uses a faster,
+        // more efficient hashing algorithm to generate the consistency checks
+        // in order to support larger files without the consistency checking process taking an incredibly long time
+        error.code === 'IMPORT_ERROR' &&
+        error.params?.some(
+          param =>
+            param.includes('INTERNAL_ERROR') &&
+            param.includes('Expected to find an inline checksum') &&
+            param.includes('.xxhash')
+        )
+      ) {
+        throw new Error('Importing the VM backup requires that CH >= 8.1 be restored.')
```

Comment:
What about throwing an [incorrectState](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-common/src/api-errors.js#L193) error instead ?

Reply:
Yes.

- `Error` are hidden from (non-admin) clients
- free messages are not easily parsed/translated

And please, log the original error somewhere to help debug.